app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -87,7 +87,7 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     final Commit commit = await datastore.lookupByValue<Commit>(commitKey);
     final CiYaml ciYaml = await scheduler.getCiYaml(commit);
     final Target target = ciYaml.postsubmitTargets.singleWhere((Target target) => target.value.name == task!.name);
-    if (task.status == Task.statusFailed || task.status == Task.statusInfraFailure) {
+    if (task.status == Task.statusFailed || task.status == Task.statusInfraFailure || _taskTimedOut(task)) {
```

Comment:
We should use `Task.statusCancelled` instead.

Reply:
Do we want to always retry if a build is canceled? Eg: If I went in and manually canceled a build, should we expect it to be retried?

Reply:
That probably isn't a reasonable use case, since that would block the tree. I've updated it to check for `Task.statusCancelled` instead!

Reply:
Thanks for the update. Yeah, post-submit CI is not expecting much human behavior, except the `manual retry`.
Some other `cancelled` cases are like `bot_died`, luci incident`, etc, and we want to treat them same as `failures` and and to auto retry to green the tree.